mirrord/cli/src/config.rs

```diff
@@ -223,6 +223,39 @@ pub(super) enum OperatorCommand {
         #[arg(short = 'f')]
         config_file: Option<String>,
     },
+    /// Operator session management commands.
+    ///
+    /// Allows the user to forcefully kill living sessions.
+    #[command(subcommand)]
+    Session(SessionCommand),
+}
+
+/// `mirrord operator session` family of commands.
+///
+/// Allows the user to forcefully kill operator sessions, use with care!
+#[derive(Debug, Subcommand, Clone, Copy)]
+pub(crate) enum SessionCommand {
+    /// Kills the session specified by `id`.
+    Kill {
+        /// Id of the session.
```

Comment:
I'd mention format, like `given as hexadecimal string`

Reply:
I'm afraid some users would put `0x` thinking it's required, as we just show them `Session ID`.

Reply:
Okok. Just leaving this without specifying format will work as well, given that operator status prints ids in hex :+1: